### PR TITLE
How to turn off error mitigation and suppression

### PR DESCRIPTION
**Pull request by beckykd**
partially address #1453

### CODE REVIEW

File: docs/run/configure-error-mitigation.mdx [hunk 1]
```diff
@@ -471,6 +471,37 @@ job = estimator.run(circuits=[psi1], observables=[H1], parameter_values=[theta1]
 psi1_H1 = job.result()
 ```
 
+<span id="no-error-mit"></span>
```
Comment (Eric-Arellano) [inline docs/run/configure-error-mitigation.mdx:474]:
```suggestion
<span id="no-error-mitigation"></span>
```

File: docs/run/configure-runtime-compilation.mdx [hunk 2]
```diff
@@ -8,7 +8,7 @@ description: How to configure runtime compilation in Qiskit Runtime.
 
 Runtime compilation techniques optimize and transform your circuit to minimize errors. Runtime compilation adds some classical pre-processing overhead to your overall runtime. Therefore, it is important to achieve a balance between perfecting your results and ensuring that your job completes in a reasonable amount of time.
 
-Primitives let you employ runtime compilation by choosing advanced runtime compilation options and, for Estimator V2, by setting the optimization level (`optimization_level`) option.
+Primitives let you employ runtime compilation by choosing advanced runtime compilation options and, for Estimator V2, by setting the optimization level (`optimization_level`) option. If you don't want any processing done to minimize errors, follow the instructions in [Turn off all error mitigation and error suppression.](configure-error-mitigation#no-error-mit)
```
Comment (Eric-Arellano) [inline docs/run/configure-runtime-compilation.mdx:11]:
```suggestion
Primitives let you employ runtime compilation by choosing advanced runtime compilation options and, for Estimator V2, by setting the optimization level (`optimization_level`) option. If you don't want any processing done to minimize errors, follow the instructions in [Turn off all error mitigation and error suppression.](configure-error-mitigation#no-error-mitigation)
```

File: docs/run/configure-error-mitigation.mdx [hunk 3]
```diff
@@ -471,6 +471,37 @@ job = estimator.run(circuits=[psi1], observables=[H1], parameter_values=[theta1]
 psi1_H1 = job.result()
 ```
 
+<span id="no-error-mitigation"></span>
+## Turn off all error mitigation and error suppression
+
+Many researchers want total control over their experiments and don't want the primitives to transform their circuits. To accomplish this, for EstimatorV2, set `resilience_level = 0` and `optimization_level = 0`. For SamplerV2, only measurement twirling is enabled by default. So one needs to set `options.twirling.enable_measure = False`.
```
Comment (jyu00) [inline docs/run/configure-error-mitigation.mdx:477]: A couple of suggestions:

1. Our built in error mitigation sounds less useful if many researchers don't want them 😛 
2. Simplify the wording to make it easier for the future if/when we enable more by default


```suggestion
You can turn off all error mitigation and suppression if you are, for example, doing research on your own mitigation techniques. To accomplish this, for EstimatorV2, set `resilience_level = 0` and `optimization_level = 0`. For SamplerV2, turn off measurement twirling.
```

File: docs/run/configure-error-mitigation.mdx [hunk 4]
```diff
@@ -471,6 +471,25 @@ job = estimator.run(circuits=[psi1], observables=[H1], parameter_values=[theta1]
...
+You can turn off all error mitigation and suppression if you are, for example, doing research on your own mitigation techniques. To accomplish this, for EstimatorV2, set `resilience_level = 0` and `optimization_level = 0`. For SamplerV2, no changes are necessary because no error mitigation or suppression options are enabled by default.
+
+Example:
+
+Turn off all error mitigation and suppression in EstimatorV2.
+
+```python
+from qiskit_ibm_runtime import EstimatorV2 as Estimator, Options
+
+options = estimator.options
```
Comment (jyu00) [inline docs/run/configure-error-mitigation.mdx:486]: How complete does the code snippet need to be? This block, for example, doesn't initialize `estimator`.

Comment (beckykd) [reply docs/run/configure-error-mitigation.mdx:486]: I suppose if we're showing the import, we should show the initialization.

**Review by jyu00 — APPROVED** [review]
